pkg/aggregate/aggregate.go

```diff
@@ -58,11 +59,51 @@ var aggrToString = map[AggrType]string{
 	aggrTypeStddev: "stddev", aggrTypeStdvar: "stdvar", aggrTypeAll: "*",
 }
 
+var aggrToSchemaField = map[string]config.SchemaField{
+	"count":  {Name: "count", Type: "array", Nullable: true, Items: "double"},
```

Comment:
I know that the list of supported aggregators is fixed right now but it could change later. I think it will be more flexible if this list will be defined in configuration.

Reply:
i would skip it now